src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1528,6 +1528,20 @@ std::unique_ptr<IParkImporter> ParkImporter::CreateS6(IObjectRepository& objectR
     return std::make_unique<S6Importer>(objectRepository);
 }
 
+static void show_error(uint8_t errorType, const rct_string_id& errorStringId)
```

Comment:
Why did you choose to use ```const rct_string_id&``` here? You should pass small things like integers always by value, this is not performance critical code but its generally better to do.

Reply:
I can see it being simpler to read and since the copy for ints is negligible, but is there another reason why it's better to pass by value? Also do you think the function should be inline?